pkgs/by-name/af/afflib/package.nix

```diff
@@ -31,8 +32,11 @@ stdenv.mkDerivation rec {
     openssl
     python3
   ]
-  ++ lib.optionals (with stdenv; isLinux || isDarwin) [ fuse3 ]
-  ++ lib.optionals stdenv.hostPlatform.isDarwin [ libiconv ];
+  ++ lib.optionals stdenv.hostPlatform.isLinux [ fuse3 ]
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    fuse
+    libiconv
```

Comment:
I think you can drop the libiconv dependency. It doesn't appear to be linked by libafflib.dylib.